next/components/ui/InbaFeaturedArticles/InbaFeaturedArticles.tsx

```diff
@@ -0,0 +1,79 @@
+import { InbaArticleEntityFragment } from '@backend/graphql'
+import BlogPostCard from '@components/molecules/presentation/BlogPostCard'
+import BlogPostHorizontalCard from '@components/molecules/presentation/BlogPostHorizontalCard'
+import BlogPostImageCard from '@components/molecules/presentation/BlogPostImageCard'
+import ResponsiveCarousel from '@components/organisms/Carousel/ResponsiveCarousel'
+import { generateImageSizes } from '@utils/generateImageSizes'
+import { getNumericLocalDate } from '@utils/local-date'
+import { useTranslations } from 'next-intl'
+import React from 'react'
+
+export type InbaFeaturedArticlesProps = {
+  articles: InbaArticleEntityFragment[]
+}
+
+export const InbaFeaturedArticles = ({ articles }: InbaFeaturedArticlesProps) => {
```

Comment:
Please add a comment, that this component is a copy of other component, just changed for inba article.

Maybe it also make sense to add a note to the original component. 

It's mostly for further changes and refactoring, that the developer should check/change also the other component.